src/test/java/me/duncte123/loadingbartestclass/Main.java

```diff
@@ -40,7 +40,8 @@ public static void main(String[] args) {
 
         LoadingBarConfig config = LoadingBarConfig.defaultConfig()
                 .setFillColor(Color.RED)
-                .setBorderColor(Color.PINK);
+                .setBorderColor(Color.PINK)
+                .setPrecision(3);
```

Comment:
Please re-render the example with this config

Reply:
The new rendered file is not detected as a changed file